spec/controllers/admin/email_templates_controller_spec.rb

```diff
@@ -0,0 +1,215 @@
+require 'rails_helper'
+
+describe Admin::EmailTemplatesController do
+
+  it "is a subclass of AdminController" do
+    expect(Admin::EmailTemplatesController < Admin::AdminController).to eq(true)
+  end
+
+  context "while logged in as an admin" do
+    before do
+      @user = log_in(:admin)
+    end
+
+    after do
+      UserHistory.delete_all
```

Comment:
Hmm do we need this here? The transaction should be rollbacked after each test.

Reply:
Not really. It just was the easiest way to solve a particular problem in a test. 😉 I'm going to change it.